lib/active_shipping/carriers/fedex.rb

```diff
@@ -662,10 +664,12 @@ def parse_tracking_response(response, options)
           description = event.at('EventDescription').text
           type_code = event.at('EventType').text
 
-          time          = Time.parse(event.at('Timestamp').text)
-          zoneless_time = time.utc
+          timestamp          = event.at('Timestamp').text
+          time               = Time.parse(timestamp)
+          timezone_ambiguous = (timestamp =~ TIMEZONE_OFFSET_FROM_DATE_REGEX).blank?
+          zoneless_time      = time.utc
 
-          shipment_events << ShipmentEvent.new(description, zoneless_time, location, description, type_code)
+          shipment_events << ShipmentEvent.new(description, zoneless_time, location, description, type_code, zone_ambiguous)
```

Comment:
Pick one or the other. `zone_ambiguous` or (preferably) `timezone_ambiguous`. because here you reference a variable that doesn't exist.